graders/python/python_autograder/code_feedback.py

```diff
@@ -252,10 +269,10 @@ def check_numpy_array_allclose(
     @classmethod
     def check_list(
         cls,
-        name,
-        ref,
-        data,
-        entry_type=None,
+        name: str,
+        ref: list,
+        data: list | None | Any,
+        entry_type: Any | None = None,
         accuracy_critical=False,
         report_failure=True,
     ):
```

Comment:
Missing return type

Reply:
Resolved in [194175a](https://github.com/PrairieLearn/PrairieLearn/pull/11122/commits/194175a4f9352bc3916a3d96d0e3611680716aec).